Resources/public/js/views/services/plugins/ez-versionsplugin.js

```diff
@@ -132,6 +136,101 @@ YUI.add('ez-versionsplugin', function (Y) {
         },
 
         /**
+         * deleteVersionDraft event handler, opens confirm box to confirm that selected draft
+         * are going to be deleted
+         *
+         * @method _deleteDraftConfirm
+         * @private
+         * @param {EventFacade} e deleteVersionDraft event facade
+         */
+        _deleteDraftConfirm: function (e) {
+            var service = this.get('host');
+
+            service.fire('confirmBoxOpen', {
+                config: {
+                    title: "Are you sure you want to remove selected drafts?",
+                    confirmHandler: Y.bind(function () {
+                        this._deleteVersion(e.versions, e.afterDeleteVersionsCallback);
+                    }, this),
+                    cancelHandler: Y.bind(e.afterDeleteVersionsCallback, this, false)
+                }
+            });
+        },
+
+        /**
+         * Removes given versions. After that, calls the callback function.
+         *
+         * @method _deleteVersion
+         * @protected
+         * @param {Array} versions array containing eZ.VersionInfo objects for removal
+         * @param {Function} callback
+         */
+        _deleteVersion: function (versions, callback) {
+            var that = this,
+                service = this.get('host'),
+                content = service.get('content'),
+                notificationIdentifier = 'delete-versions-' + content.get('id') + '-' + versions.length,
+                countRemovedVersions = 0,
+                countRemovedVersionsFails = 0,
+                tasks = new Y.Parallel();
+
+            this._notify(
+                "Removing versions for '" + content.get('name') + "'",
+                notificationIdentifier,
+                'started',
+                5
+            );
+
+            Y.Array.each(versions, function (version) {
+                var end = tasks.add(function (error, response) {
+                        if (error) {
+                            countRemovedVersionsFails++;
+                            return;
+                        }
+
+                        countRemovedVersions++;
+                    });
+
+                version.destroy({remove: true, api: service.get('capi')}, end);
+            });
+
+            tasks.done(function () {
+                var errorNotificationIdentifier, successNotificationIdentifier,
+                    versionsRemoved = (countRemovedVersions > 0);
+
+                if (countRemovedVersions === versions.length) {
```

Comment:
@dpobel the behavior is copy pasted from remove location. The goal behind this is to know which notification replaces which. Especially to handle the case where only one (or a few) deletion have errors but not all of them. So we need to display both error and success messages. Do you think we should simplify it ?